tob-web/src/app/general-data.service.ts

```diff
@@ -342,53 +354,50 @@ export class GeneralDataService {
   }
 
   translateClaimDescription(credTypeId, claimName, defVal?) {
```

Comment:
This function was updated to pipe the observable, and to use the proper data structure to get the label.